public/js/admin.js

```diff
@@ -659,6 +659,17 @@ var sortAccountsTable = function(type) {
     });
 }
 
+//toggles the view of Visiblility button in the Question-Table View
+var toggleButtonVisibility = function(){
+    if (document.getElementById('vis').value === 'true') {
```

Comment:
seems odd that true and false are between quotes

Reply:
@esteeroramy it doesn't really matter, it was just used as a flag since we had to put a string value there.

Reply:
Removed the quotes